index.html

```diff
@@ -0,0 +1,89 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>SimCom Documentation</title>
+  </head>
+  <body>
+    <!-- <nav class="sticky--nav">
+
+      
+      <h3 class="hero">SimCom</h3>
+      <ul class="doc--nav--list">
+        <li class="doc--nav--items"><a href="#home">home</a></li>
+        <li class="doc--nav--items"><a href="./pages/components.html">docs</a></li>
+        <li class="doc--nav--items"><a href="#connect">connect</a></li>
+      </ul>
+    </nav> -->
+
+    <nav class="sticky--nav">
+      <h3 class="nav--hero">SimCom</h3>
+      <ul class="nav--list">
+        <li class="nav--items"><a href="#home">home</a></li>
+        <li class="nav--items"><a href="/pages/components.html">docs</a></li>
+        <li class="nav--items"><a href="#connect">about</a></li>
+      </ul>
+    </nav>
+
+    <!-- header  section -->
+    <header id="home" class="doc--section home">
+      <h1>
+        Super easy to set up and get started with. Lightweight, customizable and
+        easy to extend. Have a consistent look and feel for all your web apps.
+        Just add the below stylesheet in your main CSS file to get started with.
+      </h1>
+    </header>
+
+    <!-- main section -->
+    <main id="docs" class="docs">
+      <iframe
+        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=night-owl&wt=none&l=javascript&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%2522https%253A%252F%252Fsimcom.netlify.app%252Fstyle.css%2522%29%253B"
+        style="
+          width: 1024px;
+          height: 473px;
+          border: 0;
+          transform: scale(1);
+          overflow: hidden;
+          background-color: white;
+        "
+        sandbox="allow-scripts allow-same-origin"
+      >
+      </iframe>
+
+      <button class="button primary--button go--to--doc">
+        <a href="/pages/components.html">go to docs</a>
+      </button>
+    </main>
+    <!-- footer -->
+    <footer id="connect" class="connect">
+      <h2>connect socially</h2>
+      <ul class="bullet-none">
+        <li>
+          <a target="blank" href="https://twitter.com/jay_r30"
+            ><i class="fa fa-twitter"></i
+          ></a>
+        </li>
+        <li>
+          <a target="blank" href="https://github.com/jay-rajput30"
+            ><i class="fa fa-github"></i
+          ></a>
+        </li>
+        <li>
+          <a
+            target="blank"
+            href="https://in.linkedin.com/in/jay-rajput-23304bab"
+            ><i class="fa fa-linkedin"></i
+          ></a>
```

Comment:
as you're using { fa fa-github } icons from font awesome, please ask user to include in there project folder otherwise it will break